CMakeLists.txt

```diff
@@ -581,6 +581,12 @@ install(
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/applications
 )
 
+install(
+    FILES
+    ${CMAKE_SOURCE_DIR}/freeorion.appdata.xml
+    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/appdata
```

Comment:
While the Fedora page that you linked to does appear to  (rather indirectly to my eyes) refer to this file as being installed in this appdata directory,  both the freedesktop.org page that you linked (and which the Fedora page itself links to) and the debian page that I linked both appear to me to explicitly direct that the file be installed to the /usr/share/metainfo/ directory.